CONTRIBUTING.md

```diff
@@ -1,167 +1,113 @@
+* [Create a new branch](https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/).
+* Navigate to the `ajax/libs` directory.
+* Click on "Create new file".
+* For the file name, enter `LIBRARY_NAME/package.json`. LIBRARY_NAME will be the name of the directory in `ajax/libs`. For example, `jquery/package.json`.
+* Compose a valid `package.json`file following the conventions listed above. Use minified versions for `filename` in `package.json` even if they don't exist. Our maintainers will minify the library when hosting it. For example, `jquery.min.js`.
+* Make a new commit and submit the pull request.
 
-## B. pull request
+#### Pre-Flight Checks
 
-### a. Sparse checkout and shallow clone
-* [The short document of how to use sparse checkout and shallow clone is here](https://github.com/cdnjs/cdnjs/blob/master/documents/sparseCheckout.md)
-* Sparse checkout and shallow clone can totally raise efficient of working with cdnjs repo and avoid the problem of case-insensitive filesystem, for detail reason, see it below.
-* The minor reason that you should use sparse checkout and shallow clone:
-  * There is no need to clone the whole history of cdnjs repo if you are not tracing issues or maintaining this project, and there is no need to checkout all the libs on cdnjs for the same reason(unless you want to access the libs without the Internet).
-* And the major reason:
-  * The size of the git repo behind cdnjs is about 2.7GB, but the extracted files will use about 70GB space on your filesystem. If you are using case-insensitive filesystem(usually on Windows and MacOS), you may have problem with [**S**ortable](https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/Sortable) and [**s**ortable](https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/sortable), we have an issue disscussion here: [#3650](https://github.com/cdnjs/cdnjs/issues/3650)
+* Have you complied with all our conventions?
+* Have you followed the library's directory structure?
+* Does a valid and accurate `package.json` file exist for the library?
+* Have you minified CSS and JavaScript assets?
+* Did all tests pass when you ran `npm test`?
 
+#### Miscellaneous Information
 
+* We recommend working on a Unix-like environment (for example, GNU/Linux or BSD distributions except macOS) due to how git works.
+* If this document didn't answer a question of yours, refer to [CONTRIBUTING-WIP.md](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING-WIP.md) (might be outdated). In case of any conflicts, follow CONTRIBUTING.md. Also, open an issue and report this if you can.
+* New contribution documentation is in the works and will be released soon.
+* Add auto-update configuration for all libraries if it doesn't exist. This is irrespective of whether you are updating or adding the library.
+* Pro tip: Use `git diff` before creating to commit to see the changes you have made. You can also use `git log -p` or `git show sha1hash` to differentiate.
```

Comment:
"creating to commit to see the change"  grammar change into:-  "creating commit to see the change"